### PR TITLE
Added Forex market hour pages

### LINKED ISSUE #1199: Missing Forex Market Hours Pages
#### Expected Behavior

The [Writing Algorithms > Securities > Asset Classes > Forex > Market Hours](https://www.quantconnect.com/docs/v2/writing-algorithms/securities/asset-classes/forex/market-hours) contains sub-pages for each Forex pair, like how the [CFD Market Hours](https://www.quantconnect.com/docs/v2/writing-algorithms/securities/asset-classes/cfd/market-hours) pages are set up.

#### Actual Behavior

The Forex Market Hours page just shows the generic Forex market hours. We don't have documentation on the market hours for each Forex Pair. The market hours of all the Forex pair used to be the same, until [LEAN PR 6998](https://github.com/QuantConnect/Lean/pull/6998/files#diff-b42ba09194308ece58b2283b0da71cf4855d2124ac73ab336706d0931f9b2330).

#### Checklist

- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

### PR DESCRIPTION
**Pull request by LouisSzeto**
#### Description
Added generic and special subpairs' market hour pages.

#### Related Issue
closes #1199

#### Motivation and Context
Market hour changes (https://github.com/QuantConnect/Lean/pull/6998)

#### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [X] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:2]: Let's replace "contracts" with "pairs".

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:9]: Let's replace this table for a list. Both of these column pretty much contain the same information

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:16]: Is this table missing some pairs? [Supported Assets](https://www.quantconnect.com/docs/v2/writing-algorithms/datasets/oanda/forex-data#05-Supported-Assets) shows more pairs.

If the remaining pairs use the "generic" market hours, we should add a sentence underneath the table that explains it.

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/11 EURNZD/01 Introduction.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/11 EURNZD/01 Introduction.html:1]: "contract" => "pair"

**Review by DerekMelchin — CHANGES_REQUESTED** [review]

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (AlexCatarino) [reply 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:16]: The problem is the text above: "_The following table shows the available contracts in the OANDA Forex market. Some of these contracts may have different trading periods than the overall OANDA Forex market._" which is not correct. It should be something like: "_The following pairs have different trading periods than the overall Forex market._"

We shouldn't use "/" to delimit the currencies since they are never referred to like this in Lean. Also, showing that "GBPNZD" represents "GBP/NZD" doesn't add useful information.

File: code-generators/market-hour-convertor.py
Comment (LouisSzeto) [reply code-generators/market-hour-convertor.py:9]: Maybe I'll wait for #1200 to be merged, then I rebase this PR with edits.

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (LouisSzeto) [reply 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:16]: `GBP/NZD` this slashed representation is the actual instrument name in OANDA (see https://github.com/QuantConnect/Lean/blob/master/Data/symbol-properties/symbol-properties-database.csv and https://www.oanda.com/sg-en/trading/cfds/forex/)

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html
Comment (LouisSzeto) [reply 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html:9]: I replied in the next comment

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/03 Market Opening Hours.html
Comment (LouisSzeto) [reply 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/03 Market Opening Hours.html:1]: This kinda pending on #1206. For now I'll manually change it first until #1206 is merged.

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/09 Supported Assets.html:2]: Replace "." => ":"

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/09 Supported Assets.html:8]: These two columns are the same. Let's just use a list instead

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 EURNZD/01 Introduction.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 EURNZD/01 Introduction.html:1]: "contract" => "pair"

File: 03 Writing Algorithms/03 Securities/99 Asset Classes/11 CFD/04 Market Hours/09 Supported Assets.html
Comment (DerekMelchin) [inline 03 Writing Algorithms/03 Securities/99 Asset Classes/11 CFD/04 Market Hours/09 Supported Assets.html:2]: "." => ":"

**Review by DerekMelchin — CHANGES_REQUESTED** [review]

**Pull request merged**
